lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -104,6 +104,7 @@ type VicContainerProxy interface {
 	Signal(vc *viccontainer.VicContainer, sig uint64) error
 	Resize(id string, height, width int32) error
 	Rename(vc *viccontainer.VicContainer, newName string) error
+	Remove(vc *viccontainer.VicContainer, removeVols bool) error
```

Comment:
Since we are using the docker types here in the container proxy we should use `types.ContainerRmConfig` instead of creating `removeVols`. Since it contains all of the options from a docker personality perspective for a `docker rm` situation.

Reply:
Fixed - thanks 👍